scenarios/Loadfilename_as_column_gcs/gcs_filename_add.dig

```diff
@@ -0,0 +1,26 @@
+timezone: Asia/Tokyo
+_export:
+  gcs:
+    bucket: td_test_bucket
+    path_prefix: files
+    upload_path_prefix: with_filename
+    datafile: data.tmp
+  td:
+    database: database_name
+    table: master_with_filename
+    endpoint: api.treasuredata.com
+
++read_gcsfiles:
+  docker:
+    image: "digdag/digdag-python:3.9"
```

Comment:
👍

Reply:
I checked that it can run without any problems.